app/Traits/CallsBucketsTrait.php

```diff
@@ -52,7 +52,19 @@ public function signedUrl(string $file_path, $asset_id, int $transaction)
                 'private_key' => storage_path('app/' . config('filesystems.disks.cloudfront.secret')),
                 'expires'     => Carbon::now()->addDay()->timestamp,
             ];
-            return $client->getSignedUrl($request_array);
+
+            $signed_url = $client->getSignedUrl($request_array);
+            $query_parameters_string = parse_url($signed_url, PHP_URL_QUERY);
+            $query_parameters = [];
+            parse_str($query_parameters_string, $query_parameters);
+            $key = $this->getKey();
+
+            if (!empty($key)) {
+                $signature = isset($query_parameters['Signature']) ? $query_parameters['Signature'] : $signedUrl;
+                \Log::channel('cloudfront_api_key')->notice($key . ' ' . $signature);
```

Comment:
was this going to change to info, or did you handle it a different way?

Reply:
I have handled it directly into the custom log handler (`CloudfrontApiKeyFormatter`). In this class I have set a custom log format to remove the severity level value. The custom format is: `"%datetime% %message% %context% %extra%\n"`